ts/src/sourcemapper.ts

```diff
@@ -0,0 +1,213 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Since this file is largely copied from the Debug agent, please keep the copyright message identical. More specifically, do not adjust the year.

Does it make sense to spin this code into a standalone module? /cc @DominicKramer

Reply:
Shifted this to third_party.